test/declarative_test_fwk/runner.py

```diff
@@ -0,0 +1,142 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Runner for Declarative tests framework scenarios for custom resources
+"""
+
+from declarative_test_fwk import model, helper
+import pytest
+import sys
+import logging
+from acktest.k8s import resource as k8s
+
+
+def run(scenario: model.Scenario) -> None:
+    """
+    Runs steps in the scenario
+    """
+    if not scenario:
+        return
```

Comment:
Not necessary, it's not an optional parameter?

Reply:
Removed this check.